.github/workflows/lint.yml

```diff
@@ -0,0 +1,14 @@
+name: Lint Markdown
+on:
+  pull_request:
+    types: [opened,synchronize]
+    
+jobs:
+  lint:
+    runs-on: macos-latest
```

Comment:
why on macos? Just curious

Reply:
So that we don't need to use setup-ruby action. Ruby is pre-installed on macOS.

Reply:
The problem is each account get a certain amount of github action credits, a OS X runner consume 10x the amount of credits than a linux one. I'd really rather not use OS X ones

Reply:
@ripienaar  😆 that limit is for private repositories only. You can check it [here](https://github.com/features/actions)

Reply:
I used actions recently on [thispsj/test-free-for-dev](https://github.com/thispsj/test-free-for-dev) and [thispsj/actions-lab](https://github.com/thispsj/actions-lab). But see my billing here :

![Billing](https://user-images.githubusercontent.com/51746608/132324169-f546db3c-db4f-4715-89e2-377ef3cb1583.png)

Reply:
I have many repos :)

Reply:
So ?

Reply:
It won't count @ripienaar .

Reply:
Do these actions not run in my account?

Reply:
@ripienaar they do but the billing is incurred only for private repositories. You're free to use as many minutes on public repositories (although GitHub has abuse systems to catch up with users who abuse this free limit)

Reply:
Correct, I already use 100s of minutes and will soon start using many 100s more as I move some projects from CircleCI. Not all accounts are the same

Reply:
So @ripienaar is this file ok now ?

Reply:
It cant be mac os

Reply:
@ripienaar why ?

Reply:
My account includes a total limit of x thousand minutes, I already use 100s of minutes and will start using many 100s more.  A Mac runner will consume 10 x as many minutes as a Linux one and so I might soon run into issues.

These minutes are not free nor are they unbilled, even in public repos.  So I dont want to end up in a situation where I cannot do my actual work becuase we use a os x runner here.

Reply:
@ripienaar I'm not understanding. There is no hard limit for public repositories.

[Please read this](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions)

![Actions are free](https://user-images.githubusercontent.com/51746608/132328415-0c612b7a-7481-48e3-ad46-41f09318d0ce.png)